.github/workflows/ci.yaml

```diff
@@ -1,73 +1,58 @@
 name: ci
 on:
   push:
-    branches: [main]
-  pull_request:
-    branches: [main]
-  schedule:
-    - cron: '15 22 * * *'
-  workflow_dispatch: {} # support manual runs
-# Prevent writing to the repository using the CI token.
-# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
-permissions: read-all
```

Comment:
It looks like this got dropped. Does some build step need write access to the project?

Reply:
Ah, good catch. This shouldn't need write permissions, no. I meant to change this to remove all permissions. Took me a while to find a guide on these but `contents: read` should be enough for cloning and building.

Reply:
If we're not going to use `read-all`, can you update the other connect-* repos to match?

Reply:
I can try, but this permission setup should be a workflow by workflow thing and I'm not sure if I'm the best person to make that judgement for all connect workflows.